opportunities_test.rb

```diff
@@ -0,0 +1,39 @@
+When /^I am on the "([^"]*)" Application as a "([^"]*)"$/ do |application, user|
+
+     step %{I am logged in as "#{user}" user}
+     App.power_nap(App::MIN_SLEEP)
+     step %{I click on app_launcher icon}
+     App.power_nap(App::MIN_SLEEP)
+     step %{I click on "#{application}"}
+end
+
+And /^I am on the Opportunities homepage$/ do
+  step %{I click on app_launcher icon}
+  on(Opportunities_testPage).text_tag("search_module","Opportunities")
```

Comment:
Please use the existing OpportunitiesPage page object to include the elements related to Opportunity screen.